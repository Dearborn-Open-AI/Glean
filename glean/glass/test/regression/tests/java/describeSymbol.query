action: describeSymbol
args: "test/java/com/facebook/glean/glean/Glean"
