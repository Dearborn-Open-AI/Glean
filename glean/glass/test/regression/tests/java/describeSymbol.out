[
    "@generated",
    {
        "annotations": [
            {
                "name": "Deprecated.java.lang",
                "source": "Deprecated"
            }
        ],
        "comments": [],
        "contains_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": true,
            "hasMoreParents": false
        },
        "extends_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "kind": 7,
        "language": 5,
        "location": {
            "filepath": "fbcode/glean/lang/java-alpha/indexer/examples/java/com/facebook/glean/glean/Glean.java",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 8,
                "lineEnd": 26
            },
            "repository": "test"
        },
        "modifiers": [],
        "name": {
            "container": "com.facebook.glean.glean",
            "localName": "Glean"
        },
        "pretty_comments": [],
        "repo_hash": "testhash",
        "signature": "public class Glean",
        "sym": "test/java/com/facebook/glean/glean/Glean",
        "sym_location": {
            "filepath": "fbcode/glean/lang/java-alpha/indexer/examples/java/com/facebook/glean/glean/Glean.java",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 8,
                "lineEnd": 26
            },
            "repository": "test"
        },
        "sym_other_locations": [],
        "type_xrefs": [
            {
                "span": {
                    "length": 5,
                    "start": 13
                },
                "type": "test/java/com/facebook/glean/glean/Glean"
            }
        ],
        "visibility": 20
    }
]